resources/ory/templates/priority-class.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: {{ .Values.priorityClassName }}
+value: 2000000
```

The one that is actually deployed on the cluster is kyma-priority-high
```suggestion
value: 2100000
```